papers/hatra21/hatra21.tex

```diff
@@ -5,6 +5,9 @@
 \acmYear{2021}
 \acmConference[HATRA '21]{Human Aspects of Types and Reasoning Assistants}{October 2021}{Chicago, IL}
 \acmBooktitle{HATRA '21: Human Aspects of Types and Reasoning Assistants}
+\acmDOI{}
+\acmISBN{}
+\expandafter\def\csname @copyrightpermission\endcsname{\raisebox{-1ex}{\includegraphics[height=3.5ex]{cc-by}} This work is licensed under a Creative Commons Attribution 4.0 International License.}
 
 \usepackage{listings}
```

Comment:
prune this?

Reply:
We need this if we want to CC-BY license the paper. By default ACM grabs the copyright.

Reply:
Sorry, this was aimed at `\usepackage{listings}`, not the copyright addition above